.maintain/playbooks/templates/nginx/composable-sandbox.conf.j2

```diff
@@ -0,0 +1,112 @@
+server {
+    listen 80;
+    server_name {{ domain }};
+    location /.well-known/acme-challenge/ {
+        root /var/www/certbot;
+    }
+    location / {
+        return 301 https://$host$request_uri;
+    }
+}
+
+server {
+    listen 9901 ssl http2;
+    server_name {{ domain }};
+
+    ssl_certificate /etc/letsencrypt/live/{{ domain }}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/{{ domain }}/privkey.pem;
+
+    # Various TLS hardening settings
+    # https://raymii.org/s/tutorials/Strong_SSL_Security_On_nginx.html
```

Comment:
not sure if we need so secured nginx for tests, but ok.
why do we need ngingx? to have tls rermination? please comment on that in playbook of test

Reply:
so arhitecture (layers, depoyment) of test would be clear to QA and other devs

Reply:
Added README.md with details.